pyiron_gui/project/project_browser.py

```diff
@@ -154,14 +363,21 @@ def _busy_check(self, busy=True):
 
     def _update_files(self):
         # HDF and S3 project do not have list_files
+        node_filter = ['NAME', 'TYPE', 'VERSION', 'HDF_VERSION']
+        file_ext_filter = ['.h5', '.db']
+        self.nodes = self.project.list_nodes()
+        if self._is_pyiron_obj:
+            self.nodes = [node for node in self.nodes if node not in node_filter]
+        self.dirs = self.project.list_groups()
         self.files = list()
         if self._show_files:
             try:
                 self.files = self.project.list_files()
             except AttributeError:
                 pass
-        self.nodes = self.project.list_nodes()
-        self.dirs = self.project.list_groups()
+        self.files = [file for file in self.files if not
+                      (file in [node + '.h5' for node in self.nodes] and file.endswith(tuple(file_ext_filter)))
+                      ]
```

Comment:
Nit) I am suspicious codacy will complain. At least I would prefer either 

```python
foo = [x for x in
       bar]
```

or (my personal top choice when one-liner won't work)

```python
foo = [
    x for x in
    bar
]
```

but not

```python
foo = [x for x in
       bar
      ]
```